tests/sentry/uptime/endpoints/test_organization_uptime_stats.py

```diff
@@ -188,23 +193,28 @@ def test_no_uptime_subscription_id(self):
             resolution="1d",
         )
         assert response.status_code == 400
+        assert response.json() == "No project uptime subscription ids provided"
 
-    @skip("vgrozdanic: This test is flaky and should be skipped")
     def test_too_many_periods(self):
-        """Test that the endpoint returns data for a simple uptime check."""
-
+        """
+        Test that requesting a high resolution across a large period of time
+        produces a 400 response.
+        """
         response = self.get_response(
             self.organization.slug,
             project=[self.project.id],
             projectUptimeSubscriptionId=[str(self.project_uptime_subscription.id)],
             since=(datetime.now(timezone.utc) - timedelta(days=90)).timestamp(),
             until=datetime.now(timezone.utc).timestamp(),
-            resolution="1h",
+            resolution="1m",
```

Comment:
This was the main fix, I just set the resolution way lower so we will absolutely hit the period limit